website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx

```diff
@@ -121,103 +130,184 @@ function SubmissionDetailsPage() {
     setChange(false);
   };
 
+  const mediaTypeIconMap = {
+    image: <img src={imgIcon} alt="Pic icon" className="media-type-icon" />,
+    video: <img src={vidIcon} alt="Video icon" className="media-type-icon" />,
+    audio: <img src={audIcon} alt="Audio icon" className="media-type-icon" />,
+  };
+
+  const getStatusImage = (status) => {
+    switch (status) {
+      case 'Approved':
+        return approvedDot;
+      case 'Incomplete':
+        return incompleteDot;
+      case 'Rejected':
+        return rejectedDot;
+      default:
+        return null;
+    }
+  };
+
   return (
     <Box sx={{ display: 'flex', backgroundColor: '#F8F8F8' }}>
       <Box sx={{
-        width: '60%', marginLeft: '5%', marginTop: '5%', marginBottom: '5%', marginRight: '1%', padding: '2%', backgroundColor: '#FFFFFF',
+        width: '60%', marginLeft: '5%', marginTop: '5%', marginBottom: '5%', marginRight: '1%', padding: '4%', backgroundColor: '#FFFFFF', boxShadow: 1, borderRadius: 2,
       }}
       >
-        <Box sx={{ display: 'flex' }}>
-          <Typography variant="h6">
+        <Box sx={{ display: 'flex', alignItems: 'center' }}>
+          <h1 className="title">
             Submission Details
-          </Typography>
+          </h1>
           <Box sx={{ marginLeft: 'auto' }}>
-            { (edit) ? (
+            {edit ? (
               <>
               </>
             ) : (
-              <Button sx={{ backgroundColor: '#4C4C9B' }} onClick={() => setEdit(true)} variant="contained">Edit</Button>
+              <button type="button" onClick={() => setEdit(true)} style={{ border: 'none', background: 'transparent', cursor: 'pointer' }}>
+                <img src={editIcon} alt="Pencil" className="edit-icon" />
+              </button>
             )}
           </Box>
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Date Submitted:
-          </Typography>
-          <Typography>
+          </p>
+          <p className="details-text">
             {details.date}
-          </Typography>
+          </p>
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Art Title:
-          </Typography>
+          </p>
           {(edit) ? (
             <TextField
               value={title}
               onChange={(e) => setTitle(e.target.value)}
+              inputProps={{
+                style: {
+                  color: '#53595C',
+                  border: '0.75px solid #8A9195',
```

Comment:
Not sure why this is happening in my version but let me know
[](url)
<img width="512" alt="Screenshot 2023-07-20 at 11 50 20 PM" src="https://github.com/lablueprint/skid-row-arts-alliance/assets/62224382/649e35a1-1eb3-41ce-8920-0f11fc3c87dd">

Reply:
@he-is-james I'm unable to recreate this issue on my end. Is it still happening?